comms.ino

```diff
@@ -588,7 +797,7 @@ void sendToothLog(bool useChar)
           Serial.write(lowByte(tempToothHistory[x]));
         }
       }
-      //Serial.flush();
+      Serial.flush();
```

Comment:
Any reason this has been reenabled? Forcing the serial flush can cause some large latency problems is all.